connector/sccconnector/README.md

```diff
@@ -0,0 +1,61 @@
+# Semantic Convention Checker Connector
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aconnector%2Fscc%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aconnector%2Fscc) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aconnector%2Fscc%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aconnector%2Fscc) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@madvikinggod](https://www.github.com/madvikinggod), [@codeboten](https://www.github.com/codeboten) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] | [Stability Level] |
+| ------------------------ | ------------------------ | ----------------- |
+| traces | logs | [development] |
+| metrics | logs | [development] |
+| logs | logs | [development] |
+
+[Exporter Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#exporter-pipeline-type
+[Receiver Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#receiver-pipeline-type
+[Stability Level]: https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+<!-- end autogenerated section -->
+
+This is a connector for the Collector to continuously monitor telemetry's adherence to a semantic convention.
+
+To use a collector with this built in add this to your config:
+
+```
+connectors:
+  scc:
```

Comment:
any thoughts around naming this `semconvchecker` or something a bit more descriptive? i worry that scc is hard to understand for someone not familiar with the acronym and sccconnector is a lot of C's in a row

Reply:
Sure, So this would then be the `semconvcheckerconnector`?  I can work on getting that done asap.

Reply:
@codeboten I've moved it all to semconvchecker.  Anything else?